desktop-exporter/app/routes/trace-view.jsx

```diff
@@ -40,22 +44,29 @@ function WaterfallView(props) {
     );
 }
 
-function DetailView() {
+function DetailView(props) {
     return (
         <div className='detail'>
-            Details will go here
+            Name: {props.spanData.name} <br />
+            Kind: {props.spanData.kind} <br />
+            Start: {props.spanData.startTime} <br />
+            End: {props.spanData.endTime} <br />
         </div>
-    )
+    );
 }
 
 export default function TraceView() {
     const traceData = useLoaderData();
+    const [activeIndex, setActiveIndex] = React.useState(0);
```

Comment:
Instead of relying on the ordering of the `spans` array, I think it will be clearer in the long-run to save the `spanID` that we're highlighting directly instead.

```suggestion
    const [selectedSpanID, setSelectedSpanID] = React.useState(traceData.spans[0].spanID);
```